crm_claim/report/crm_claim_report.py

```diff
@@ -159,12 +159,13 @@ def init(self):
          """
 
         tools.drop_view_if_exists(self.env.cr, self._table)
-        self.env.cr.execute("""
+        sql = """
             CREATE OR REPLACE VIEW %s AS (
                 %s
                 from
                 %s
                 %s
             )""" % (
             self._table, self._select(), self._from(), self._group_by(),
-        ))
+        )
+        self.env.cr.execute(sql)
```

Comment:
I see you're trying to trick the linter.

Actually you can use this instead, which would be the correct way:


    from psycopg2.extensions import AsIs
    self.env.cr.execute("""
        CREATE OR REPLACE VIEW %s AS (
                 %s
                 from
                 %s
                 %s
             )""", (
             AsIs(self._table), AsIs(self._select()), AsIs(self._from()), AsIs(self._group_by()),
    ))

Reply:
Thank you @Yajo  :)